python/python/tests/test_async_client.py

```diff
@@ -907,6 +908,34 @@ async def test_rpushx(self, redis_client: TRedisClient):
         with pytest.raises(RequestError):
             await redis_client.rpushx(key2, [])
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_linsert(self, redis_client: TRedisClient):
+        key1 = get_random_string(10)
+        key2 = get_random_string(10)
+
+        assert await redis_client.lpush(key1, ["4", "3", "2", "1"]) == 4
+        assert await redis_client.linsert(key1, InsertPosition.BEFORE, "2", "1.5") == 5
+        assert await redis_client.linsert(key1, InsertPosition.AFTER, "3", "3.5") == 6
+        assert await redis_client.lrange(key1, 0, -1) == [
+            "1",
```

Comment:
Don't trust person whom you stole that test from!

Reply:
Can't tell if you're joking haha. Took it from your PR here:
https://github.com/Bit-Quill/glide-for-redis/pull/176/files#diff-5db9a3bd41d647cfdba521ea248ea07d7c78bc9aaa41a26bb923d4f92a3efdd2R1223

Reply:
I know, don't trust me